### DISCUSSION TITLE
Help understanding if this is a bug or a feature I can't understand

### DISCUSSION THREAD

**Discussion by vncsalencar** [paperwm/PaperWM#957] [Q&A] [answered]
Whenever I hit the Super/Win button to get to the overview, I can see three "workspace miniatures" on each monitor (I have a main one and one to the left of it)
But on the left monitor, the leftmost workspace is selected while in the main one it's the middle workspace
So on normal view I can see windows that are on both middle workspaces but when on overview, the left monitor switches to the leftmost workspace and shows nothing cause i have nothing on it...
I don't understand if this is a configuration or feature I just didn't understand or if it's a bug

**Comment by Lythenas** [comment, marked as answer]
Yes it is a bug. In short this is an incompatibility between how PaperWM uses Workspaces and how Gnome normally uses workspaces. The Gnome overview is mostly useless currently.

I believe there are some issues with discussions about this. If I remember correctly we would need to implement a custom overview and replace the gnome one. But that was never a priority.

Personally I basically never use the overview and I disabled opening it when I press the super key.
